packages/containers/src/actionAPI.js

```diff
@@ -10,6 +10,14 @@ export function getActionsProps(context, ids, model) {
 		tmpIds = [ids];
 	}
 
+	// in the case of ConfirmDialog, the component is loaded (and hidden) even when not
+	// used at the beginning,
+	// so ids (state.validateAction) is neither a string neither an array in this case
+	// as it takes a default state of the container which is an object.
+	if (!Array.isArray(tmpIds)) {
+		return ids;
+	}
```

Comment:
the comment is really not appropriate.
you have passed an object as id. this is not the way to call it.